index.html

```diff
@@ -1,52 +1,314 @@
-    </main>
+	<main class="container">
+		<header class="header">
+			<nav class="navigation">  
+				<div class="logo"> 
+					<a href="#">George M'sapenda</a>
+				</div>
+					<button class="btn">
+						<img src="/images/Icon - Menu.png" alt="menu bar icon">
+					</button>                    
+				</div>
+			</nav>
+		</header>
+
+			<section class="intro">
+					<h1 class="intro_greeting">I'm George<br> Nice to see you!</h1>
+					<p class="intro_text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+					<div class="connect">
+						<div class="connect_text">
+							<p>LET'S CONNECT<p>    
+						</div>
+					</div>
+					<ul class="socials">
+						<li>
+							<a href="https://twitter.com/GeorgeMsapenda" target="_blank">
+								<img src="images/Twitter icon.png" alt="Twitter link">
+							</a>
+						</li>
+						<li>
+							<a href="linkedin.com/in/george-m-sapenda-593750201" target="_blank">
+								<img src="images/Linkedin icon.png" alt="Linkedn link">
+							</a> 
+						</li>
+							<li>
+								<a href="https://github.com/c00p75" target="_blank">
+									<img src="images/Github icon.png" alt="Github link">
+								</a>
+						</li>                                
+					</ul>
+			</section>
+
+			<section class="Works">
+				<div class="work_item">
+					<img src="images/Snapshoot Portfolio.png" alt="snapshot portifolio image">
+					<div class="work_details">
+						<div class="project">
+							<p class="work_item_header">Tonic</p>
+							<div class="work_history">
+								<p class="client">
+									<span class="client_name">Canopy</span>
+								</p>
+								<p class="counter">
+									<img src="images/Counter.png" alt="counter icon">                            
+								</p>
+								<p class="role">
+									<span class="role_name">Back End Dev</span>
+								</p>
+								<p class="counter">
+									<img src="images/Counter.png" alt="counter icon">                            
+								</p>
+								<p class="year">
+									<span class="date">2015</span>
+								</p>
+							</div>   
+						</div>       		
+						<p class="work_summary">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+						<div class="tags">
+							<p class="html">
+								<span>html</span>
+							</p>
+							<p class="css">
+								<span>css</span>
+							</p>
+							<p class="js">
+								<span>javascript</span>
+							</p>
+						</div>
+						<div class="project_link">
+							<p>
+								<a href="#">See Project</a>
+							</p>
+						</div>
+					</div> 
+				</div>
+
+				<div class="work_item">
+					<img src="images/Snapshoot Portfolio (1).png" alt="snapshot portifolio image">
+					<div class="work_details">
+						<div class="project">
+							<p class="work_item_header">Multi-Post Stories</p>
```

Comment:
:red_circle: Please make sure you use `hx` tag for All the project card headers. A `p` tag is not semantically appropriate. Please change it to a heading